axelor-human-resource/src/main/java/com/axelor/apps/hr/service/lunch/voucher/LunchVoucherMgtService.java

```diff
@@ -33,6 +33,9 @@
   @Transactional(rollbackOn = {Exception.class})
   public void validate(LunchVoucherMgt lunchVoucherMgt) throws AxelorException;
 
+  @Transactional(rollbackOn = {Exception.class})
+  public void close(LunchVoucherMgt lunchVoucherMgt) throws AxelorException;
```

Comment:
Please add javadoc and remove `Transactional` occurences in the interface

Reply:
Done